modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/SettingsImpl.kt

```diff
@@ -999,7 +1001,17 @@ class SettingsImpl @Inject constructor(
     }
 
     override fun getAutoPlayNextEpisodeOnEmpty(): Boolean {
-        return getBoolean(Settings.PREFERENCE_AUTO_PLAY_ON_EMPTY, false)
+        val defaultValue = when (Util.getAppPlatform(context)) {
+            AppPlatform.Automotive -> true
+            AppPlatform.Phone -> true
```

Comment:
The default value for phone was `false` earlier. Can you reconfirm if we're making it `true` now?

Reply:
In this PR it is true, but this is more of a placeholder. Apparently the end goal is for it to be true for new installations, and false for users upgrading the app. That will be handled in a separate PR though.